From 310b305b10ac8bdeaf8684ea078a532c54d5340e Mon Sep 17 00:00:00 2001
From: Kazuya Mizuguchi <kazuya.mizuguchi.ks@renesas.com>
Date: Tue, 15 Nov 2016 17:03:46 +0900
Subject: [PATCH 1479/2066] arm64: dts: r8a7795-es1: Fix phy-mode to
 rgmii-txid for EthernetAVB

The clock internal delay mode setting of EthernetAVB is changed as follows.
This patch fixes phy-mode from rgmii-id to rgmii-txid, because dts file is
set to enable only tx clock internal delay mode and it may fail with 1Gbps
communication.

phy-mode    : before      : after
----------------------------------
"rgmii-id"  : tx delay    : tx and rx delay
"rgmii-rxid": not support : rx delay
"rgmii-txid": not support : tx delay

Signed-off-by: Kazuya Mizuguchi <kazuya.mizuguchi.ks@renesas.com>
Signed-off-by: Takeshi Kihara <takeshi.kihara.df@renesas.com>
[zou:Original patch taken from
git://git.kernel.org/pub/scm/linux/kernel/git/horms/renesas-bsp.git
 v4.9/rcar-3.5.1]
Signed-off-by: zou cao <cao.zou@windriver.com>
---
 arch/arm64/boot/dts/renesas/r8a7795-es1.dtsi |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/arch/arm64/boot/dts/renesas/r8a7795-es1.dtsi b/arch/arm64/boot/dts/renesas/r8a7795-es1.dtsi
index aef2167..2fab335 100644
--- a/arch/arm64/boot/dts/renesas/r8a7795-es1.dtsi
+++ b/arch/arm64/boot/dts/renesas/r8a7795-es1.dtsi
@@ -1119,7 +1119,7 @@
 					  "ch24";
 			clocks = <&cpg CPG_MOD 812>;
 			power-domains = <&sysc R8A7795_PD_ALWAYS_ON>;
-			phy-mode = "rgmii-id";
+			phy-mode = "rgmii-txid";
 			#address-cells = <1>;
 			#size-cells = <0>;
 		};
-- 
1.7.5.4

